index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="icon" href="Tesla.svg">
+        <link rel="stylesheet" href="style.css" />
+        <title>Elon Musk Tribute</title>
+    </head>
+    
+    <body class="main-container">
+        <div id="main">
+            <h1 id="title" class="heading">Elon Musk</h1>
+            <div class="img-div" id="img-div">
+                <p id="img-caption" class="description">Below is an image of Elon Musk in front of a Tesla Sign</p>
+        <img id="image" class="image" src="musk-tesla.jpg" alt="Picture of Elon Musk"
+          width="800px"/>
+        <p class="tribute-info" id="tribute-info">
+          Elon Musk is a tech billionaire from South Africa. He is famouse for
+          starting the electric car company Tesla. He also founded Spacex, a
+          space exploration company that has pioneered some of the greatest
+          leaps in techology and space exploration.
+        </p>
+        <h1>You can find out more about Tesla at their website: </h1>
+          <h1 class="link" id="tribute-link-tesla" href="https://www.tesla.com/">Tesla's website</h1>
```

Comment:
The right way to use a link in `html` we use the `<a>` tag, the `<h1>` tag is for the title.
When we need to use the link we put it in the `<a>` that contains the `href` attribute, which indicates the link's destination as used here. So instead of `<h1>` use `<a>` tag.